complement/pipeline.yml

```diff
@@ -0,0 +1,38 @@
+steps:
+  - command:
+      - "env"
```

Comment:
these are probably redundant?

Reply:
Probably. I cribbed it from [dendrite's pipeline](https://github.com/matrix-org/pipelines/blob/master/dendrite/pipeline.yml), and the result we get is [this](https://buildkite.com/matrix-dot-org/dendrite/builds/3908#79ed3bb4-35d5-49b4-aeeb-5af7e6e64602/48-61).

Perhaps useful to see `GOPATH` and that `GOROOT` is not set, though I don't expect any of those environment variables to ever really change.